buildSrc/src/main/java/com/uber/okbuck/core/dependency/exporter/JsonDependencyExporter.java

```diff
@@ -38,10 +48,11 @@ public void export(Set<ExternalDependency> dependencies) {
             .collect(Collectors.toSet());
 
     Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    String file = exportDependenciesExtension.getFile();
```

Comment:
we could create the Path object here which can be used in `createParent` and `Files.newBufferedWriter`